src/modules/lyrics/providers.js

```diff
@@ -155,16 +155,29 @@ BetterLyrics.LyricProviders = {
 
     const data = await response.json();
 
-    if (data && data.syncedLyrics && typeof data.duration === "number") {
-      BetterLyrics.Utils.log(BetterLyrics.Constants.LRCLIB_LYRICS_FOUND_LOG);
-      return {
-        lyrics: BetterLyrics.LyricProviders.parseLRC(data.syncedLyrics, data.duration),
-        source: "LRCLib",
-        sourceHref: "https://lrclib.net/",
-      };
-    } else {
-      throw new Error(BetterLyrics.Constants.NO_LRCLIB_LYRICS_FOUND_LOG);
+    if (data) {
+      // First try synced lyrics
+      if (data.syncedLyrics && typeof data.duration === "number") {
+        BetterLyrics.Utils.log(BetterLyrics.Constants.LRCLIB_LYRICS_FOUND_LOG);
+        return {
+          lyrics: BetterLyrics.LyricProviders.parseLRC(data.syncedLyrics, data.duration),
+          source: "LRCLib",
+          sourceHref: "https://lrclib.net/",
+        };
+      }
+      // Fall back to plain lyrics if available
+      else if (data.plainLyrics) {
+        BetterLyrics.Utils.log(BetterLyrics.Constants.LRCLIB_LYRICS_FOUND_LOG + " (plain lyrics)");
+        return {
+          lyrics: BetterLyrics.LyricProviders.formatPlainLyrics(data.plainLyrics),
+          source: "LRCLib (Plain)",
+          sourceHref: "https://lrclib.net/",
+          isPlainLyrics: true,
```

Comment:
Do we need this flag to work correctly? The code that checks if all the lyrics have zero detection should be able to pick up on these lyrics without this flag.